### ISSUE TITLE
"VAC was unable to verify you game session" multiple times because of charlatano?

### ISSUE THREAD

**Issue by edgelife10** [Jire/Charlatano#913]
I have had this error multiple times in a day. Everytime i verify integrity of game files and restart steam it solves the issue but again after one match (with charlatano on) same error occurs. i even reinstalled steam and csgo but after two matches with chrlatano on again the same error appeared. i am just quitting chralatano for few days till i get some update. 
Anyone else havingthis issue or knows something about it?

ps: 1. i know this error has nothing to do with VACban, and i have tried all the methods to fix the error including fresh install of steam and game.
2. this error doesnt occur as long as chralatano is not used after verifying the game files.

**Comment by LinuxNoneWindow** [comment]
restart your computer its working fine for me

**Comment by effingnoone** [comment]
i have even reinstalled the steam and game 2 times. still issue not resolved.

**Comment by Bobby8904** [comment]
Works for me never received those issues yet past week of using id say uninstall or dont use CCleaner and other cleaners it can fuck up the vac on steam giving you the vac error

**Comment by Zyphix** [comment]
@effingnoone  You are right this has nothing to do with the cheat or anything related to actual suspicion of cheats. This has always been a known problem to CSGO. I used to get this error a lot in past years and I figured out it was giving the error because I had 2 anti viruses installed and both running
